libafl_qemu/librasan/asan/Cargo.toml

```diff
@@ -34,6 +34,8 @@ guest = []
 hooks = []
 ## Enable support for shadow memory and tracking in the host
 host = ["dep:syscalls"]
+## Intialize all allocations with 0xff
+initialize = []
```

Comment:
maybe a dumb question. why this is a feature
shouldn't all allocations be always initialized with 0xff? to indicate that every thing is addressable?

Reply:
When you call malloc etc, there's no guarantee what the contents of the memory will be. To provide deterministic behaviour, the memory can be set to 0xff with librasan. However, this comes with a performance cost. Hence why it is optional. This relates to the user buffer itself. Not the shadow memory representing it.

Reply:
ok i though it was about shadow mem

Reply:
No worries. Thanks for the merge!